scripts/deploy/localstack/stopContainer.sh

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022-2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker stop sleeper-localstack
+echo "Stopped localstack container"
```

Comment:
If you've started it with `docker-compose up`, you can stop it with `docker-compose down`.